### ISSUE TITLE
Error: The client is missing a required right

### ISSUE THREAD

**Issue by mpek** [sarkinios/sarkui#3]
When starting Sarkui.exe
Error: The client is missing a required right

![NVIDIA_Share_dTDEDHNoWr](https://user-images.githubusercontent.com/6974695/197020857-11b882bd-6d47-4903-99cc-d261c7bf8d22.png)

System is a German Windows 11 22H2 Build 22621.675

**Comment by sarkinios** [comment]
SarkUI tested and works on win 10 but check if you have made any changes in your system rights. Right click / Properties/ Security tab and check your permissions

**Comment by lallasgr** [comment]
On mine SarkUi works fine. English language.

https://i.imgur.com/qehba21.png

Do you have any restrictions on exe files under specific user? Are you the admin?

**Comment by mpek** [comment]
The message is gone after a big cleanup. Now SarkUI does not start. 
AT the 1st start, a different message from Win 11 appeared, saying that it is potentionally dangerous, but this is only because you did not pay MS, and I wanted to start it anyway. 
But SarkUI still does not open. 
I checked my firewall and every time i want to start SarkUI, my firewall blocks wermgr.exe to send a request to MS. 
So, how do I start SarkUI without sending it to MS?

**Comment by sarkinios** [comment]
What do you mean "I did not pay MS"?  Sarkui is open source. Try install .NET Frameworks, sarkui is made in Visual Studio C# NET . wermgr is part of error reporting service because the app crash in your Windows for some reason I guess.

**Comment by mpek** [comment]
That box i mentioned disappears only if you pay MS to be verified...
.NET Framework 3.5 and 4.8 Advanced Services are already installed

**Comment by sarkinios** [comment]
Are you sure you have the sarktools and lib folder in the exe directory? I suggest try to download 2nd Sarkui Package from release files [here](https://github.com/sarkinios/sarkui#release-files) which includes all folders and programs in the path.

**Comment by mpek** [comment]
Yes, I'm sure. SarkUI.v1.0.2wTools.zip with 154 MB. I put the SarkUI folder into root of C:
Now I moved the SarkUI to c:\Users\myuser\Downloads\ but no difference. Nothing appears.
